spec/integ/matrix-to-irc.spec.js

```diff
@@ -392,7 +392,7 @@ describe("Matrix-to-IRC message bridging", function() {
                 expect(client.nick).toEqual(testUser.nick);
                 expect(client.addr).toEqual(roomMapping.server);
                 expect(channel).toEqual(roomMapping.channel);
-                expect(text).toEqual(`<${repliesUser.nick}> "This" <- Reply Text`);
+                expect(text).toEqual(`<${repliesUser.nick}> "This..." <- Reply Text`);
```

Comment:
Looks like a reply to me but I trust you :)

Reply:
It is :) I sneaked in a bugfix [here](https://github.com/matrix-org/matrix-appservice-irc/pull/1465/files#diff-1461a1072635fdffa4ecfbfebe8aad698b2c3a406dd690a9546ff1f047fed580L1076) that makes the event cache actually remember the entire contents rather than just the first line – that's why this not includes the `...`, since it now properly remembers that the message that we're replying to didn't end there